pkg/gds/emails/client_test.go

```diff
@@ -56,8 +56,19 @@ func TestClientSendEmails(t *testing.T) {
 	err = models.SetContactVerification(vasp.Contacts.Technical, "", true)
 	require.NoError(t, err)
 
-	require.NoError(t, email.SendVerifyContacts(vasp))
-	require.NoError(t, email.SendReviewRequest(vasp))
-	require.NoError(t, email.SendRejectRegistration(vasp, "this is a test rejection from the test runner"))
-	require.NoError(t, email.SendDeliverCertificates(vasp, "testdata/foo.zip"))
+	sent, err := email.SendVerifyContacts(vasp)
+	require.NoError(t, err)
+	require.Equal(t, 1, sent)
+
+	sent, err = email.SendReviewRequest(vasp)
+	require.NoError(t, err)
+	require.Equal(t, 1, sent)
+
+	sent, err = email.SendRejectRegistration(vasp, "this is a test rejection from the test runner")
+	require.NoError(t, err)
+	require.Equal(t, 1, sent)
+
+	sent, err = email.SendDeliverCertificates(vasp, "testdata/foo.zip")
+	require.NoError(t, err)
+	require.Equal(t, 1, sent)
```

Comment:
I can't quite remember what's in our test example, but we might want to test to make sure that if there is a technical and say billing contact, that only the technical one gets it. And maybe we should double check that sent is not ever greater than 1 (but it shouldn't be).

Reply:
I've added a test to make sure that only one contact gets the email; but it's tougher to test who got sent the email since the function doesn't return that email. We'd have to mock the email backend (which might be a good idea anyway) to determine the destination. Hopefully, the test I just pushed will hold us over for now.